tests/execute/results-custom/test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    testName="/test/custom-results"
+    rlPhaseStartTest "${testName}"
+        rlRun "run=\$(mktemp -d)" 0 "Creating run direcotory/id for ${testName}"
+        rlRun -s "tmt run -a --id ${run} test --name ${testName} provision --how local execute -vv report -vv" 1 "Run test providing results by itself"
+        rlAssertGrep "00:11:22 pass /test/custom-results/test/passing" $rlRun_LOG
+        rlAssertGrep "00:22:33 fail /test/custom-results/test/failing" $rlRun_LOG
+        rlAssertGrep "         pass /test/custom-results/test/no_keys \[1/1\]" $rlRun_LOG
+        rlAssertGrep "total: 2 tests passed and 1 test failed" $rlRun_LOG
+        rm $rlRun_LOG
+        rlRun "rm -r ${run}" 0 "Remove run directory"
+    rlPhaseEnd
+
+    testName="/test/missing-custom-results"
+    rlPhaseStartTest "${testName}"
+        rlRun "run=\$(mktemp -d)" 0 "Creating run direcotory/id for ${testName}"
+        rlRun -s "tmt run -a --id ${run} test --name ${testName} provision --how local execute -vv report -vv" 2 "Run test providing results by itself"
+        rlAssertGrep "Custom results file '/tmp/.*/plans/default/execute/data/test/missing-custom-results/data/results.yaml' not found." $rlRun_LOG
+        rm $rlRun_LOG
+        rlRun "rm -r ${run}" 0 "Remove run directory"
+    rlPhaseEnd
+
+    testName="/test/empty-custom-results-file"
+    rlPhaseStartTest "${testName}"
+        rlRun "run=\$(mktemp -d)" 0 "Creating run direcotory/id for ${testName}"
+        rlRun -s "tmt run -a --id ${run} test --name ${testName} provision --how local execute -vv report -vv" 3 "Run test providing results by itself"
```

Comment:
Also duplicated many times, what about introducing the following variable?

    command="tmt run -a --id ${run} ${testName} provision --how local execute -vv report -vv test --name"

And using it in all commands?